Remove auto-index/category docs views/URLs (#5781)
newrelic/docs-website: src/content/docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations.mdx

## Requirements [#permissions]

> Check the docs for our [Azure integrations](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list) for requirements for specific integrations.

Reviewer: ```suggestion
See [Azure integrations](/docs/infrastructure/microsoft-azure-integrations/azure-integrations-list) for requirements on specific integrations.
```
